Sources/SpeziOpenAI/MessageView.swift

```diff
@@ -67,13 +68,9 @@ public struct MessageView: View {
         }
     }
     
-    
-    /// - Parameters:
-    ///   - chat: The chat message that should be displayed.
-    ///   - hideSystemMessages: If system messages should be hidden from the chat overview.
```

Comment:
Please ensure that you leave and have documentation for any `public` method. This makes it easier for someone to dig through the code and the documentation on the Swift Package index to find and understand the method they are looking for.

You can e.g., reuse some wording from `Sources/SpeziOpenAI/MessagesView.swift` (e.g., for the `hideMessagesWithRoles` parameter)